deploy/ceph-conf.yaml

```diff
@@ -14,6 +14,10 @@ data:
     auth_cluster_required = cephx
     auth_service_required = cephx
     auth_client_required = cephx
+    # log_to_stderr = true  # log command stderr in in csi-* container logs
+    # debug_rbd = 30  # enable librbd logs in rbd container logs
+    # debug_rados = 30  # enable librados logs in rbd/cephfs container logs
```

Comment:
rbd/cephfs/nfs container logs

Reply:
Tweaking the comment to this, which is more adequate:
```
    # enable ceph librbd,librados logs in rbd/cephfs/nfs container logs
    # log_to_stderr = true
    # debug_rbd = 30  # enable debug rbd logs
    # debug_rados = 30  # enable debug rados logs
    # debug_rbd_mirror = 30  # enable debugging logs for rbd mirroring daemon
```